content/en/docs/concepts/security/pod-security-admission.md

```diff
@@ -37,9 +37,7 @@ To use this mechanism, your cluster must enforce Pod Security admission.
 
 ### Built-in Pod Security admission enforcement
 
-In Kubernetes v{{< skew currentVersion >}}, the `PodSecurity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-is a beta feature and is enabled by default. You must have this feature gate enabled.
-If you are running a different version of Kubernetes, consult the documentation for that release.
+From Kubernetes v1.23, the `PodSecurity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is a beta feature and is enabled by default. This page is part of the documentation for Kubernetes v{{< skew currentVersion >}}. If you are running a different version of Kubernetes, consult the documentation for that release.
```

Comment:
Please wrap the long lines properly.

```suggestion
From Kubernetes v1.23, the `PodSecurity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
is a beta feature and is enabled by default. This page is part of the documentation for Kubernetes
v{{< skew currentVersion >}}.
If you are running a different version of Kubernetes, consult the documentation for that release.
```

Reply:
Thanks, @tengqm for looking at this, I will update it.

Reply:
Hi, @tengqm I have addressed the comment please take a look.